packages/suite/src/actions/wallet/__tests__/blockchainActions.test.ts

```diff
@@ -1,12 +1,20 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Move this file to the common package later.

Reply:
Btw there is one mock in tests (`src/middlewares/suite/__tests__/buttonRequestMiddleware.test.ts`) using **@wallet-actions/blockchainActions** which does not exist anymore => it will fail.

Reply:
Fixed (removed)